app/javascript/app/components/prep-link/prep-link-component.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { PureComponent } from 'react';
+import prepLogo from 'assets/icons/prep-logo.svg';
+import Icon from 'components/icon';
+import styles from './prep-link-styles.scss';
+
+class PrepLink extends PureComponent {
```

Comment:
I don't like the idea of creating a component just for that... at least we could create a component that accepts the image and content as props or include it in the poverty folder.
But having a general component just for this I don't think is very useful.